apix_documentation_swagger_v3.yaml

```diff
@@ -1175,6 +1202,14 @@ paths:
                     state:
                       type: string
                       example: "NY"
+                status:
+                  type: string
+                  description: Only needed when you want to create a transaction
+                    that is to be processed at a later date. For this purpose,
+                    the value needs to be `on_hold`; if any other value is sent,
+                    the transaction will be created with the `initialized`
+                    status.
```

Comment:
I think we should mention how to process the `on_hold` transaction in the future by updating the status to `initialized` from `on_hold`.

Reply:
Oh, you did it later. my bad. Looks good 👍